### ISSUE TITLE
NullPointerException changeVisibleCellViewsBackgroundForColumn

### ISSUE THREAD

**Issue by Swat24** [evrencoskun/TableView#256]
Hi,
when tap on header of table without data i getting NullPointerException 
file: SelectionHandler
line: 341

this:
```
            AbstractViewHolder holder = (AbstractViewHolder) cellRowRecyclerView
                    .findViewHolderForAdapterPosition(column);
```

cellRowRecyclerView is null

**Comment by MGaetan89** [comment]
This seem to be fixed in `master` (at least after #268 is merged).

**Comment by MGaetan89** [comment]
@Swat24 the fix was merged in `master`. Can you close this issue if it works for you?
